light/client.go

```diff
@@ -7,6 +7,8 @@ import (
 	"fmt"
 	"time"
 
+	format "github.com/ipfs/go-ipld-format"
+	dag "github.com/ipfs/go-merkledag"
```

Comment:
why even alias this? This will shadow the variable `dag` and `merkledag` is more descriptive (and would be the default without that alias).

Reply:
I think IDE did that, will change in something more decriptive

Reply:
Removed the alias